server/core/templates/user_settings_partial_base.html

```diff
@@ -1,10 +1,14 @@
+(% macro side_menu_item(label, href, uri, svg) %)
+<a hx-select="main" hx-target="main" hx-swap="outerHTML show:false swap:1s" href="(( href ))" 
+    class="list-group-item list-group-item-action(% if let Some(path_and_query) = uri.path_and_query() %)(% if path_and_query.to_string().starts_with(href) %) active(% endif %)(% endif %)">
+    (( svg|safe ))(( label ))
+</a>
+(% endmacro %)
+
 <main class="container-xxl pb-5">
     <div class="d-flex flex-sm-row flex-column">
-        <div class="list-group side-menu">
-            <a href="#" class="list-group-item list-group-item-action active">
-                <svg xmlns="http://www.w3.org/2000/svg" width="20" height="20" fill="currentColor" class="bi bi-person-fill me-3" viewBox="0 0 16 16">
-                    <path d="M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6"/>
-                </svg>Profile</a>
+        <div class="list-group side-menu flex-shrink-0">
+            (% call side_menu_item("Profile", "/ui/profile", uri, "<svg xmlns=\"http://www.w3.org/2000/svg\" width=\"20\" height=\"20\" fill=\"currentColor\" class=\"bi bi-person-fill me-3\" viewBox=\"0 0 16 16\"><path d=\"M3 14s-1 0-1-1 1-4 6-4 6 3 6 4-1 1-1 1zm5-6a3 3 0 1 0 0-6 3 3 0 0 0 0 6\"/></svg>") %)
```

Comment:
Seems like someone's already been doing it, but please don't include inline SVG like this, please make them static files. We also need to be able to identify where they came from for licensing purposes.

Reply:
This was my doings, they're bootstrap icons under MIT https://github.com/twbs/icons